tests/jsx_quotes/jsfmt.spec.js

```diff
@@ -0,0 +1,2 @@
+run_spec(__dirname, ["flow", "typescript"], { jsxSingleQuote: true });
+run_spec(__dirname, ["flow", "typescript"], { jsxSingleQuote: false });
```

Comment:
Please add `"babylon"` to both of these tests — it’s actually a different parser from Flow’s parser.